torch/csrc/inductor/aoti_package/model_package_loader.cpp

```diff
@@ -63,6 +63,11 @@ const std::string k_separator = "\\";
 const std::string k_separator = "/";
 #endif
 
+bool startsWith(const std::string& str, const std::string& prefix) {
```

Comment:
C10 has a shim for this already.

Reply:
Or better yet: https://github.com/pytorch/pytorch/blob/main/c10/util/string_view.h